internal/fs/local_file_test.go

```diff
@@ -113,11 +113,15 @@ func (t *LocalFileTest) validateObjectNotFoundErr(fileName string) {
 	ExpectTrue(errors.As(err, &notFoundErr))
 }
 
-func (t *LocalFileTest) closeFileAndValidateObjectContents(f **os.File, fileName string, contents string) {
+func (t *LocalFileTest) closeLocalFile(f **os.File) error {
 	err := (*f).Close()
-	AssertEq(nil, err)
 	*f = nil
+	return err
+}
 
+func (t *LocalFileTest) closeFileAndValidateObjectContents(f **os.File, fileName string, contents string) {
```

Comment:
why do you need double pointer here?

Reply:
for test cleanup, we need to mark *t.f1, *t.f2 as nil, hence we are passing a reference to pointer here. 
example: https://stackoverflow.com/a/8769095/14369827